src/timetable/types.ts

```diff
@@ -0,0 +1,75 @@
+import { PopoverV2TriggerType } from "../popover-v2";
+import { ResourceScheme } from "../theme";
+
+export const ROW_BAR_COLOR_SEQUENCE = [
+    "#FFE6BB",
+    "#D8EFEB",
+    "#E6EAFE",
+    "#FAE4E5",
+    "#D3EEFC",
+] as const; // Assert to be a readonly tuple
+export const ROW_CELL_GAP = 2;
+export const ROW_INTERVAL = 15;
+export type RowBarColors = (typeof ROW_BAR_COLOR_SEQUENCE)[number];
+export type CellType = "filled" | "blocked" | "default";
+
+interface EmptyContentProps {
+    description: string;
+    illustrationScheme: ResourceScheme;
+}
+
+export interface RowData {
```

Comment:
public types need to be named more explicitly to avoid collision

```suggestion
export interface TimeTableRowData {
```